forbered_afskrivining_af_foraeldede_sagsomkostninger/excel_process.py

```diff
@@ -0,0 +1,130 @@
+            continue
+
+
+        # Step 4: Slet rækker hvor indholdsart er på listen (e.g. BØVO.)
+        if row[header_row.index('Indholdsart')] in REMOVE_INDHOLDSART:
+            continue
+
+        # Step 5: Delete rows where Aftale type is not empty
+        if row[header_row.index('Aftale type')] is not None:
+            continue
+
+        # Step 6: Skip.
+
+        # Step 7: Move rows with ZGBY or ZREN to sagsomkostninger
+        if row[header_row.index('Hovedtransakt.')] in ('ZGBY', 'ZREN'):
+            sagsomkostninger.append(row)
+            continue
+
+        # Step 8: Move rows with Ratespecifikation LRT to sagsomkostnigner.
+        if row[header_row.index('Ratespecifikation')] == 'LRT':
+            sagsomkostninger.append(row)
+        else:  # Step 9: remaining rows, I.E. all, where Ratespecifikation is not LRT goes to hovedstole.
+            hovedstole.append(row)
+
+        # exit loop
+
+    # Step 10: Delete row from sagsomkostninger if "RykkespærÅrsag" is 'N'.
+    sagsomkostninger = [row for row in sagsomkostninger if row[header_row.index('RykkespærÅrsag')] != 'N']
+
+    # Step 11: Delete row from sagsomkostninger if Forældelse is None or later than today.
+    def _expirey_date_passed(row):
+        date = row[header_row.index('Forældelsesdato')]
+        if date is None:
+            return False
+
+        if date > datetime.datetime.today():
+            return False
+
+        return True
+
+    sagsomkostninger = [row for row in sagsomkostninger if _expirey_date_passed(row)]
+
+    # Step 12: Split rows with "Indholdsart" ['DAGI', 'DAG2', 'SFO2'] to special_content_type_rows and not_special[...]
+    special_content_type_rows = []
+    not_special_content_type_rows = []
+
+    for row in sagsomkostninger:
+        if row[header_row.index('Indholdsart')] in ['DAGI', 'DAG2', 'SFO2']:
+            special_content_type_rows.append(row)
+        else:
+            not_special_content_type_rows.append(row)
+
+
+    # Step 13: select sagsomkostninger and hovedstole. match FP and Aftale
+    # create set of (FP,aftale) from hovedstole
+    # pylint: disable-next=(consider-using-set-comprehension)
+    unique = set((row[header_row.index('ForretnPartner')], row[header_row.index('Aftale')]) for row in hovedstole)
+    # remove rows from sagsomkostninger that do not have matching (FP, aftale) i hovedstole.
+    not_special_content_type_rows = [row for row in not_special_content_type_rows if
+                        (row[header_row.index('ForretnPartner')], row[header_row.index('Aftale')]) not in unique]
+
+    # Step 14: Combine lists
+    not_special_content_type_rows.extend(special_content_type_rows)
+
+    # Step 15: Delete rows where RIM Aftale == 'IN' and RIM aftalestatus == 21
+    _sagsomkostninger = []
+    for row in not_special_content_type_rows:
+        if row[header_row.index('RIM Aftale')] == 'IN' and row[header_row.index('RIM aftalestatus')] == '21':
+            continue
```

Comment:
I would negate the if statement and replace 'continue' with ' _sagsomkostninger.append(row).
It's not super important